src/components/Forms/UserEmail/index.js

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import Box from "@material-ui/core/Box";
+import { alpha, styled } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import InputBase from "@material-ui/core/InputBase";
+import FormControl from "@material-ui/core/FormControl";
+import Typography from "@material-ui/core/Typography";
+import MenuItem from "@material-ui/core/MenuItem";
+import Select from "@material-ui/core/Select";
+import OutlinedInput from "@material-ui/core/OutlinedInput";
+import useStyles from "./styles";
+
+const Input = styled(InputBase)(({ theme }) => ({
+  "label + &": {
+    marginTop: theme.spacing(3),
+  },
+  "& .MuiInputBase-input": {
+    borderRadius: 4,
+    position: "relative",
+    backgroundColor: theme.palette.mode === "light" ? "#fcfcfb" : "#fff",
+    border: "1px solid #ced4da",
+    borderRadius: 6,
+    fontSize: 16,
+    width: "100%",
+    padding: "10px 12px",
+    transition: theme.transitions.create([
+      "border-color",
+      "background-color",
+      "box-shadow",
+    ]),
+    fontFamily: ["Roboto", "Helvetica", "Arial", "sans-serif"].join(","),
+    "&:focus": {
+      boxShadow: `${alpha(theme.palette.primary.main, 0.25)} 0 0 0 0.2rem`,
+      borderColor: theme.palette.primary.main,
+    },
+  },
+}));
+
+const UserEmail = () => {
+  const classes = useStyles();
+
+  const [email1, setEmail1] = React.useState("");
+  const [email2, setEmail2] = React.useState("");
+
+  const handleChange1 = (event) => {
+    setEmail1(event.target.value);
+  };
+
+  const handleChange2 = (event) => {
+    setEmail2(event.target.value);
+  };
+
+  return (
+    <Card className={classes.card}>
+      <Box
+        component="form"
+        noValidate
+        sx={{
+          display: "flex",
+          flexDirection: "column",
+        }}
+      >
+        <Typography style={{ margin: "5px 0" }}>
+          thisismyemail@gmail.com -{" "}
```

Comment:
can you import these values from a json file and make them variables. that way another person can connect the frontend code to backend easily.